flash_examples/question_answering.py

```diff
@@ -0,0 +1,66 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from flash import Trainer
+from flash.core.data.utils import download_data
+from flash.text import QuestionAnsweringData, QuestionAnsweringTask
+
+# 1. Create the DataModule
+download_data("https://rajpurkar.github.io/SQuAD-explorer/dataset/train-v2.0.json", "./data/")
+download_data("https://rajpurkar.github.io/SQuAD-explorer/dataset/dev-v2.0.json", "./data/")
+
+datamodule = QuestionAnsweringData.from_squad_v2(
+    train_file="./data/train-v2.0.json",
+    val_file="./data/dev-v2.0.json",
+)
+
+# 2. Build the task
+model = QuestionAnsweringTask()
+
+# 3. Create the trainer and finetune the model
+trainer = Trainer(max_epochs=3, limit_train_batches=1, limit_val_batches=1)
+trainer.finetune(model, datamodule=datamodule)
+
+# 4. Answer some Questions!
+predictions = model.predict(
+    {
+        "id": ["56ddde6b9a695914005b9629", "56ddde6b9a695914005b9628"],
```

Comment:
Is it possible to not provide the id when predicting? Ideally we would just provide the context and the question. Could also be nice to add support for context + list of questions, but that could be done in a follow-up PR

Reply:
To be done in follow-up

Reply:
Currently the `id` is being used to reference the output accordingly in predict.
In a follow-up PR along with `context, list[questions]`  we can also provide non `id` based input.